nativeApp/src/screens/admin_views/AddTime/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC, useState } from 'react';
+import styled from 'styled-components/native';
+import { Form as F, Text } from 'native-base';
+
+import { Heading } from '../../../lib/ui/typography';
+import Dropdown from '../../../lib/ui/forms/Dropdown';
+import { Forms } from '../../../lib/ui/forms/enums';
+import DateTimePicker from '../../../lib/ui/forms/DateTimePicker';
+import { ColoursEnum } from '../../../lib/ui/colours';
+import HoursAndMinutesText from '../../../lib/ui/HoursAndMinutesText';
+import SubmitButton from '../../../lib/ui/forms/SubmitButton';
+
+import { getTimeDiff } from '../../../lib/utils/time';
+
+/*
+ * Types
+ */
+type Props = {
+}
+
+/*
+ * Styles
+ */
+const View = styled.View`
+  alignItems: center;
+  paddingTop: 20;
+  paddingBottom: 20;
+`;
+
+const Form = styled(F)`
+  width: ${Forms.formWidth}
+`;
+
+const Label = styled(Text)`
+  color: ${ColoursEnum.darkGrey};
+  marginTop: 15;
+`;
+
+const TimeContainer = styled.View`
+  marginTop: 15;
+`;
+
+const projects = [
+  { id: 0, name: 'General' },
+  { id: 1, name: 'Spring' },
+];
+
+const activities = [
+  { id: 0, name: 'Office work' },
+  { id: 1, name: 'Support' },
+];
+
+const volunteers = [
+  { id: 0, name: 'Kara Thrace' },
+  { id: 1, name: 'Lee Adama' },
+];
+
+
+/*
+ * Component
+ */
+const AddTime: FC<Props> = () => {
+  const [project, setProject] = useState('');
+  const [activity, setActivity] = useState('');
+  const [volunteer, setVolunteer] = useState('');
+  const [date, setDate] = useState();
```

Comment:
Think we should use `useState<Date>()` on these three, otherwise all the values typecheck as `any`.